### DISCUSSION TITLE
esse grupo está ativo?

### DISCUSSION THREAD

**Discussion by ghost** [frontendbr/forum#2403] [Perguntas e Respostas] [answered]
Oi pessoal.

Gostaria de saber se esse grupo está ativo, gostaria de contribuir com algumas ideias. esse grupo está ativo? está em funcionamento?

**Comment by felipefialho** [comment, marked as answer]
Você diz esse repositório? Está ativo sim 😄

**Reply by ghost** [reply]
Oi felipefialho, Fiz um redesign de um site de código aberto, poderia divulgar aqui?

**Reply by ninetails** [reply]
No tipo da Discussion tem uma opção para divulgar seu trabalho que creio que possa ser utilizado para isto.

![divulgue seu trabalho](https://user-images.githubusercontent.com/40065/224350589-6ad3806a-b070-4189-a791-3091b4ead64d.png)
